### PR TITLE
dash/productsjs

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: tools/gulptasks/dist-productsjs.js [hunk 1]
```diff
@@ -26,45 +24,53 @@ const TARGET_FILE = Path.join('build', 'dist', 'products.js');
...
+        (
+            'var products = ' + JSON.stringify({
+                Highcharts: { date, nr },
+                'Highcharts Stock': { date, nr },
+                'Highcharts Maps': { date, nr },
+                'Highcharts Gantt': { date, nr },
+                'Highcharts Dashboards': dashboardsProduct
+            }, void 0, '    ') + '\n'
+        )
+    );
```
Comment (goransle) [inline tools/gulptasks/dist-productsjs.js:71]: I am not sure what it's used for, but won't this overwrite the date for the Highcharts(& family) release when doing a Dashboards release?

Comment (goransle) [reply tools/gulptasks/dist-productsjs.js:71]: Oh wait, there's two tasks

**Review by sebastianbochan — APPROVED** [review]

File: tools/gulptasks/dist-productsjs.js (hunk 1 above)
Comment (bre1470) [reply tools/gulptasks/dist-productsjs.js:71]: It is a bit messy. Maybe it is better to download the latest `products.js` from code.highcharts.com and modify it?

**Review by karolkolodziej — APPROVED** [review]

**Review by goransle — APPROVED** [review]

File: tools/gulptasks/dist-productsjs.js (hunk 1 above)
Comment (bre1470) [reply tools/gulptasks/dist-productsjs.js]: What do you think about fetching the latest `products.js`?

Comment (goransle) [reply tools/gulptasks/dist-productsjs.js]: Not sure. I guess it could help avoid problems with the local dist-repo not being updated?

Comment (bre1470) [reply tools/gulptasks/dist-productsjs.js]: Yes, that was the idea. The issue though can be, that the cache on the server is not immediately updated. So releases of several products would need a time buffer in-between. Or the cache purge needs to be automated.

Comment (sebastianbochan) [reply tools/gulptasks/dist-productsjs.js]: Are we able to clear cache after release or only Sam can do it?

Comment (bre1470) [reply tools/gulptasks/dist-productsjs.js]: I know at least three more people that can do it.

**Review by pawellysy — APPROVED** [review]

**Pull request merged**